roles/kube-install/tasks/main.yml

```diff
@@ -93,6 +93,34 @@
     fi
   when: ipv6_enabled
 
+- name: Download and install kokonet-cni binary
+  block:
+    - name: Get the latest version
+      shell: >
```

Comment:
`ansible-lint` will complain about not using the `curl` module directly. Not sure if that's possible for this usecase though.

Reply:
No idea. As far as I know, no way to do that using get_url module.
Do you have a way to do that? (just a double check)